src/main/java/dumshenko/daniil/tic_tac_toe/Main.java

```diff
@@ -0,0 +1,58 @@
+package dumshenko.daniil.tic_tac_toe;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Виберіть свій символ (X або O): ");
+        char choice = scanner.nextLine().toUpperCase().charAt(0);
+
+        Player player1, player2;
+        if (choice == 'X') {
+            player1 = new Player("Player 1", PlayerType.X);
+            player2 = new Player("Player 2", PlayerType.O);
+        } else {
+            player1 = new Player("Player 1", PlayerType.O);
+            player2 = new Player("Player 2", PlayerType.X);
+        }
+
+        NewBoard board = new NewBoard(3, 3);
```

Comment:
nit: Let's name the class just `Board`. The suffix `New` is a bit of bad naming pattern for classes.